recipe/meta.yaml

```diff
@@ -52,12 +55,12 @@ test:
     - pip
 
 about:
-  home: https://www.lfd.uci.edu/~gohlke/
+  home: https://www.cgohlke.com
   summary: Laboratory for Fluorescence Dynamics (LFD) file formats
   license: BSD-3-Clause
   license_file: LICENSE
   dev_url: https://github.com/cgohlke/lfdfiles/
 
 extra:
   recipe-maintainers:
-    - goanpeca
```

Comment:
I think we shouldn't remove the original maintainer unless we get their ok to do so

Reply:
Yes do not do this. You have to ask them first.